src/mlpack/core/data/load.hpp

```diff
@@ -96,7 +96,11 @@ bool Load(const std::string& filename,
           arma::Mat<eT>& matrix,
           DatasetMapper<PolicyType>& info,
           const bool fatal = false,
-          const bool transpose = true);
+          const bool transpose = true)
+{
+  PolicyType policy;
+  return Load(filename, matrix, info, policy, fatal, transpose);
```

Comment:
thanks for the feedback!
I think the modifier in mlpack in general only returns the reference of the member value.
should I provide another `void Policy(PolicyType value)` member function for more usability?